src/magic_folder/test/fixtures.py

```diff
@@ -179,3 +186,95 @@ def _setUp(self):
             upload_dircap=self.upload_dircap,
             status=WebSocketStatusService(),
         )
+
+@attr.s
+class MagicFolderNode(object):
+    http_client = attr.ib(validator=attr.validators.instance_of(HTTPClient))
+    global_service = attr.ib(validator=attr.validators.instance_of(MagicFolderService))
+    global_config = attr.ib(validator=attr.validators.instance_of(GlobalConfigDatabase))
+
+    @classmethod
+    def create(
+        cls,
+        reactor,
+        basedir,
+        auth_token=None,
+        folders=None,
+        start_folder_services=False,
+        tahoe_client=None,
+    ):
```

Comment:
This function is taken from `magic_folder.test.test_web.treq_for_folders`, but returns an object that has more than just the HTTP client as attributes.